libs/blocks/notification/notification.js

```diff
@@ -220,6 +222,8 @@ function curtainCallback(el) {
   document.body.classList.add('mobile-disable-scroll');
   el.insertAdjacentElement('afterend', curtain);
   el.setAttribute('role', 'dialog');
+  const ariaLabel = getHeadingText(el);
+  el.setAttribute('aria-label', ariaLabel ? `${ariaLabel} Dialog` : 'Promotional Banner Dialog');
```

Comment:
NIT: `getHeadingText` returns `undefined` if there is no header, right ? So you should be able to do this
```suggestion
  el.setAttribute('aria-label', `${ariaLabel ?? 'Promotional Banner'} Dialog`);
```